### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by premgan**
Move the Oauth requirement for Ex2016 mailbox users with the supportability statement instead of "Addiitonal Information" which can be overlooked.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ For the full Microsoft Teams experience, every user should be enabled for Exchan
...
+Users hosted on either Exchange Online Dedicated - Legacy, or Exchange on-premises, must be synchronized to Azure Active Directory for Office 365. They can create and join teams and channels, add and configure tabs and bots, and chat and call. However, they can’t modify user profile pictures, or add and configure connectors. They can receive messages from connectors configured by other users. For creating and viewing meetings, it's a mixed bag: Creating and viewing meetings is supported for Exchange 2016 cumulative update 3 (CU3) and above, but not for versions prior to Exchange 2016 CU3.For Exchange on-premises, you must to configure OAuth as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://technet.microsoft.com/en-us/library/dn594521(v=exchg.150).aspx).
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md]:
```suggestion
Users hosted on either Exchange Online Dedicated - Legacy, or Exchange on-premises, must be synchronized to Azure Active Directory for Office 365. They can create and join teams and channels, add and configure tabs and bots, and chat and call. However, they can’t modify user profile pictures, or add and configure connectors. They can receive messages from connectors configured by other users. For creating and viewing meetings, it's a mixed bag: Creating and viewing meetings is supported for Exchange 2016 cumulative update 3 (CU3) and above, but not for versions prior to Exchange 2016 CU3. For Exchange on-premises, you must to configure OAuth as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://technet.microsoft.com/en-us/library/dn594521(v=exchg.150).aspx). 
```

Comment (nenonix) [inline Teams/Exchange-Teams-interact.md]:
```suggestion
Users hosted on either Exchange Online Dedicated - Legacy, or Exchange on-premises, must be synchronized to Azure Active Directory for Office 365. They can create and join teams and channels, add and configure tabs and bots, and chat and call. However, they can’t modify user profile pictures, or add and configure connectors. They can receive messages from connectors configured by other users. For creating and viewing meetings, it's a mixed bag: Creating and viewing meetings is supported for Exchange 2016 cumulative update 3 (CU3) and above, but not for versions prior to Exchange 2016 CU3.For Exchange on-premises, you must to configure OAuth as described in [Configure OAuth authentication between Exchange and Exchange Online organizations](https://technet.microsoft.com/library/dn594521(v=exchg.150).aspx). 
```

**Comment by LolaJacobsen** [comment]
@dstrome  - could you review this PR?

**Comment by scanum** [comment]
@officedocsbot assign @dstrome

**Comment by dstrome** [comment]
hi @scanum @LolaJacobsen - I'm taking a look at this issue now

**Comment by dstrome** [comment]
The existing location of the OAuth statement is correct given the context of the list it lives in. However, I'll change the heading of that list to something more descriptive of it's actual intent - to list configuration that's necessary to fully support integration between Teams and other products. I'm going to close this PR without merging and do the work in a separate branch.

**Pull request closed without merging**